crates/noirc_evaluator/src/brillig/brillig_gen/brillig_slice_ops.rs

```diff
@@ -323,11 +322,12 @@ mod tests {
     use crate::brillig::brillig_ir::tests::{create_and_run_vm, create_context};
     use crate::brillig::brillig_ir::BrilligContext;
     use crate::ssa::ir::map::Id;
+    use fxhash::FxHashMap as HashMap;
 
     fn create_test_environment() -> (FunctionContext, BrilligContext) {
         let function_context = FunctionContext {
             function_id: Id::test_new(0),
-            ssa_value_to_brillig_variable: HashMap::new(),
+            ssa_value_to_brillig_variable: HashMap::default(),
```

Comment:
Why was `new` changed to `default`?

Reply:
Ok, I see `HashMap::new` refers to `pub fn new() -> HashMap<K, V, RandomState>` which implies `RandomState` instead of the fxhash, so we can't use it here.

Reply:
The HashMap in the library is an alias for `std::collections::HashMap`, just with a different hasher. The issue seems to be `HashMap::new` in the std only creates a new hashmap for a hashmap with the RandomState hasher instead of the FxHash used in this library.